spec/model/billing_info_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require_relative "spec_helper"
+
+RSpec.describe BillingInfo do
+  subject(:bi) { described_class.create_with_id("name" => "old name") }
+
+  it "syncs with Stripe customer after create/update/destroy" do
+    allow(Config).to receive(:stripe_secret_key).and_return("secret_key")
+    expect(Stripe::Customer).to receive(:create).and_return({"id" => "cs_1234567890"})
+    expect(Stripe::Customer).to receive(:update).with("cs_1234567890", anything).twice
+    expect(Stripe::Customer).to receive(:delete).with("cs_1234567890")
+
+    expect(bi.stripe_id).to eq("cs_1234567890")
```

Comment:
We should reorder expects and bi.<ops> so that it goes like one bi.<op>, one expect. This would ensure that Stripe::Customer receives correct method call at the correct time.